docs/plugins/discovery-ec2.asciidoc

```diff
@@ -134,7 +134,11 @@ with the right signer to use.
 ==== EC2 Discovery
 
 ec2 discovery allows to use the ec2 APIs to perform automatic discovery (similar to multicast in non hostile multicast
-environments). Here is a simple sample configuration:
+environments).
+
+Note that ec2 discovery plugin expects your node running on default 9300 port.
```

Comment:
You're following a pattern, but I noticed we say `ec2`, but `GCE` below. We should probably be consistent and proper and use EC2.